src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -150,3 +150,7 @@ fluent-data-grid-cell .long-inner-content {
 .validation-message {
     color: var(--error);
 }
+
+.page-header {
+    padding: calc(var(--design-unit) * 1px) calc(var(--design-unit) * 2px);
```

Comment:
Can `grid-area: head` be on `.page-header` or is there a reason that doesn't work?